components/esp_modem/examples/simple_cmux_client/main/simple_cmux_client_main.cpp

```diff
@@ -68,16 +83,33 @@ extern "C" void app_main(void)
     assert(esp_netif);
 
 #if CONFIG_EXAMPLE_MODEM_DEVICE_BG96 == 1
-    std::unique_ptr<DCE> dce = create_BG96_dce(&dce_config, dte, esp_netif);
+    auto dce = create_SIM7070_GNSS_dce(&dce_config, dte, esp_netif);
```

Comment:
```suggestion
std::unique_ptr<DCE> dce = create_BG96_dce(&dce_config, dte, esp_netif);
```

Please revert this, or maybe
```patch
-std::unique_ptr<DCE> dce = create_BG96_dce(&dce_config, dte, esp_netif);
+auto dce = create_BG96_dce(&dce_config, dte, esp_netif);
```